imsv-docs-astro/src/content/docs/guides/core-concepts/card-funding.mdoc

```diff
@@ -60,8 +60,7 @@ Withdrawals are conducted as a two-stage process:
 1. Obtaining an Immersve-generated signature permitting a given withdrawal
 2. Executing a withdrawal and observing the effect on the funding source
 
-To begin the withdrawal process your application will
-[create a withdrawal intent](/api-reference/create-withdrawal-intent),
+To begin the withdrawal process your application will call {% link endpoint="create-withdrawal-intent" /%},
```

Comment:
Are you intentionally leaving "endpoint" out?

Reply:
yes